tools/build.sh

```diff
@@ -15,6 +15,7 @@ run_valgrind=0
 enable_java_binding=OFF
 enable_dotnet_core_binding=OFF
 enable_nodejs_binding=OFF
+enable_native_remote_modules=ON
```

Comment:
So, talked with David. This should be OFF by default since the flag is enable_native_remote. :) If we want to make ON by default we shall change the flag to disable_native_remove_modules....